packages/lesswrong/lib/collections/localgroups/newSchema.ts

```diff
@@ -155,6 +156,7 @@ const schema = {
     database: {
       type: "TIMESTAMPTZ",
       denormalized: true,
+      nullable: false,
```

Comment:
This should probably just be defaulting to the same thing as createdAt (i.e. CURRENT_TIMESTAMP), but right now it's using an `onCreate`.

Reply:
(This was already `NOT NULL` in the db, so it's not included in the migration itself.)